doc/content/verification_and_validation/ver-1dd.md

```diff
@@ -0,0 +1,49 @@
+# ver-1dd
+
+# Permeation Problem without Trapping
+
+## Test Description
+
+This verification problem is taken from [!cite](ambrosek2008verification) and builds on the capabilities verified in [ver-1d](ver-1d.md) and [ver-1dc](ver-1dc.md). However, in the current case, there are no traps in the membrane. This case is simulated in [/ver-1dd.i].
```

Comment:
```suggestion
This verification problem is taken from [!cite](ambrosek2008verification) and builds on the capabilities verified in [ver-1d](ver-1d.md) and [ver-1dc](ver-1dc.md). The configuration and modeling parameters are the same as in [ver-1d](ver-1d.md), except that, in the current case, there are no traps in the membrane. This case is simulated in [/ver-1dd.i].
```
The test description should list the values of the model parameters and case configurations, like the thickness of the plate, the diffusivity, etc. 
It is OK to say: 
`we use the same values as in [link to ver-1d],`
 but you need to make sure there is no confusion for the user.